README.md

```diff
@@ -35,30 +37,27 @@ import * as Sentry from '@sentry/browser';
 // import * as Sentry from '@sentry/react-native';
 import * as FullStory from '@fullstory/browser';
```

Comment:
not sure if you want to address this as part of the v3 upgrade, but this changed with `@fullstory/browser` v2. related PR: https://github.com/getsentry/sentry-fullstory/pull/90

Reply:
Yes let me get that in as well!